cime_config/SystemTests/rxcropmaturity.py

```diff
@@ -416,6 +454,30 @@ def _run_generate_gdds(self, case_gddgen):
             raise RuntimeError(error_message)
         self._gdds_file = generated_gdd_files[0]
 
+    def _run_interpolate_gdds(self):
+        # Save where?
+        self._gdds_file = os.path.join(self._get_caseroot(), "interpolated_gdds.nc")
+
+        # It'd be much nicer to call interpolate_gdds.main(), but I can't import interpolate_gdds.
```

Comment:
Why does

import interpolate_gdds

fail here? It's OK to call as a script, but as you say it would be nicer to call from within python.

Reply:
This is a behavior pattern I have elsewhere in rxcropmaturity.py already for reasons I can't quite remember. I think it has to do with `ctsm_pylib` not being active? I think it might be beyond the scope of the current PR (or at least my time at the moment) to figure that out, but I can add an issue about it if you'd like.

Reply:
Yeah, how about you make an issue and reference that here? We tend to now always run tests with ctsm_pylib active so that reason might have gone away. But, yeah these type of things can be sticky to solve. On the other hand it's also more robust if we call the python directly rather than go through a script called through a subprocess. So long term fixing this can save us debugging time. But, not something that needs to be solved now.

Reply:
Cool, thanks. Issue added at #2603.